nvm.sh

```diff
@@ -3590,12 +3599,16 @@ nvm() {
         if [ "${NVM_SILENT:-0}" -ne 1 ]; then
           nvm_err "N/A: version \"${PROVIDED_VERSION} -> ${VERSION}\" is not yet installed."
           nvm_err ""
-          nvm_err "You need to run \"nvm install ${PROVIDED_VERSION}\" to install it before using it."
+          if [ "${IS_VERSION_FROM_NVMRC}" != "1" ]; then
```

Comment:
i wonder if we could call nvm_ensure_version_installed here instead of duplicating this check?

Reply:
Yes, This makes sense. I have updated the pr accordingly. thanks!